packages/web/src/modules/zoom/components/controls/host-control.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback } from 'react';
+
+import { ui } from '@application';
+
+import { ZoomContextType } from '../../context';
+import { authorizeZoom } from '../../services';
+
+const { Button, Form } = ui;
+
+const HostControl = ({
```

Comment:
Add Controls to the application components namespace.

Reply:
Is this necessary if they are used only within this component?

Reply:
They won't be used inside the provider

https://github.com/MrBr/chess-tent/pull/7#discussion_r1264954800

Third components will use all these to compose them in a different places